README.md

```diff
@@ -1,6 +1,3 @@
-[![Go Report Card](https://goreportcard.com/badge/github.com/MetalBlueberry/go-plotly)](https://goreportcard.com/report/github.com/MetalBlueberry/go-plotly)
-[![godoc](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/mod/github.com/MetalBlueberry/go-plotly?tab=overview)
-
```

Comment:
Why is this removed?

Reply:
I put it back again.
I removed it as I could not use it on my fork, but since I'm merging back I added it again.